src/kessler.F

```diff
@@ -803,14 +813,16 @@ subroutine fallout_GPU(dt,train,ruh,rvh,zh,mh,mf,rain,prate,rr,rho,   &
       enddo
     enddo
     enddo
-
     !$acc end parallel
+
+    !JMD FIXME:  This loop looks incorrect. what is going on with the n-loop 
```

Comment:
How do you know that this loop is incorrect?

Reply:
This is a note to myself to come back and look at this code in more detail.  Note that the currently dx=40m namelist does not activate this code.